ui/page/components/votebar_widget.go

```diff
@@ -226,6 +227,20 @@ func (v *VoteBar) Layout(gtx C) D {
 	)
 }
 
+func (v *VoteBar) infoButtonModal() {
+	text1 := fmt.Sprintf("Total votes:  %6.0f", v.totalVotes)
+	text2 := fmt.Sprintf("Quorum requirement:  %6.0f", (v.requiredPercentage/100)*v.eligibleVotes)
+	text3 := fmt.Sprintf("Discussions:   %d comments", v.numComment)
+	text4 := fmt.Sprintf("Published:   %s", dcrlibwallet.FormatUTCTime(v.publishedAt))
+	text5 := fmt.Sprintf("Token:   %s", v.token)
+	bodyText := fmt.Sprintf("%s\n %v\n %s\n %s\n %s", text1, text2, text3, text4, text5)
+	modal.NewInfoModal(v.Load).
+		Title("Vote title").
```

Comment:
pass the proposal title to the vote bar info button and use the proposal title.

Better still just use `Proposal vote details`